src/App.css

```diff
@@ -28,6 +28,34 @@
   color: #61dafb;
 }
 
+.title {
+  padding: 24px 24px 24px 24px;
```

Comment:
the `padding` shouldn't be here, it should be included in the outermost container (the whole page) instead